relayer/src/target.ts

```diff
@@ -18,67 +19,86 @@ class Target {
     this.signer = signer;
   }
 
-  // TODO: signer should be proxy account per feed
-  // TODO: refactor using rxjs api - this.api.rx.tx...
-  private async sendBlockTx({ block, metadata }: TxData): Promise<void> {
-    const unsub = await this.api.tx.feeds
-      .put(block, metadata)
-      // it is required to specify nonce, otherwise transaction within same block will be rejected
-      // if nonce is -1 API will do the lookup for the right value
-      // https://polkadot.js.org/docs/api/cookbook/tx/#how-do-i-take-the-pending-tx-pool-into-account-in-my-nonce
-      .signAndSend(this.signer, { nonce: -1 }, ({ status, dispatchError }) => {
-        if (status.type === "InBlock") {
-          if (dispatchError) {
-            if (dispatchError.isModule) {
-              const decoded = this.api.registry.findMetaError(
-                dispatchError.asModule
-              );
-              console.log(
-                `Transaction failed: ${decoded.section}.${decoded.method}`
-              );
-            } else {
-              // Other, CannotLookup, BadOrigin, no extra info
-              console.log(`Transaction failed: ${dispatchError.toString()}`);
-            }
-          } else {
-            console.log(
-              `Transaction successful: ${polkadotAppsUrl}${status.asInBlock}`
-            );
-          }
+  private logTxResult({ status, events }: ISubmittableResult) {
```

Comment:
This should probably be a standalonefunction, it doesn't use `this`

Reply:
it will use logger after I merge another branch